docs/public-networks/how-to/monitor/understand-metrics.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Understand Besu performance metrics
+tags:
+  - public networks
+---
+
+If you've run Besu on mainnet several times with `X_SNAP` option enabled
+([see experimental options](../../reference/cli/options.md#xhelp)),
+you may have noticed graphical patterns that stand out in different metrics charts.
+
+These patterns are related to the Besu sync process.
+
+Read this page to better understand and know how to interpret these graphical patterns.
+
+## CPU Utilization
+
+The following screenshot from [monitoring Besu with Prometheus and Grafana] shows patterns related to CPU utilization.
+
+![CPU Grafana Besu dashboard patterns](../../../images/besu-cpu-pattern-during-sync.png)
+
+The CPU pattern is a "staircase" pattern, where each step represents one of the Besu running stages.
+
+### 1 -- Blocks import and world state download
+
+Step 1 highlights blocks import and world state download, two tasks that are executed in parallel in Besu.
+Besu manages these two tasks with two different pipelines.
```

Comment:
Add here : 
This step is CPU Bound because multiple threads are used to execute the stages of the two pipelines. On a VM with 8 CPU, CPU load average is about 7.5 and sometimes exceeds 10, so there is more work to do than the number of available CPUs.
<img width="1639" alt="Screenshot 2022-09-06 at 10 42 24" src="https://user-images.githubusercontent.com/5099602/188589468-24abf19a-f53f-487a-8e71-b768df914a5e.png">